special-use/special-use-POST-suds-mapping-post.json

```diff
@@ -70,14 +62,13 @@
           "startDateTime": { "type": "dateTime", "note" : "Store in middle layer."},
           "endDateTime": { "type": "dateTime", "note" : "Store in middle layer." },
           "numberParticipants" : { "type": "integer", "note" : "Store in middle layer." },
-          "AUTOPOPULATED_BY_MIDDLEWARE_NOT_INTAKE_MODULE": { "basic-api-field": "PURPOSE",
-            "note" : "This field should be a concatenation of activityDescription, locationDescription, startDateTime, endDateTime."},
-          "AUTOPOPULATED_BY_MIDDLEWARE_NOT_INTAKE_MODULE": { "basic-api-field": "crExemptInd",
-            "value" : "Y",
+          "AUTOPOPULATED_BY_MIDDLEWARE_NOT_INTAKE_MODULE": {
+            "basic-api-field": "PURPOSE",
             "basic-api-endpoint": "/contact/application",
-            "required-by-basic-api": "N"},
+            "note" : "This field should be a concatenation of activityDescription, locationDescription, startDateTime, endDateTime.",
+            "required-by-basic-api": "Y"},
```

Comment:
I think we should be consistent between `"Y"` and `"N"` or `"yes"` and `"no"`. Or actually just use `true`.